data/repositories.xml

```diff
@@ -84,4 +84,7 @@
     <phar alias="sw-cli-tools" composer="shopwarelabs/sw-cli-tools">
         <repository type="github" url="https://api.github.com/repos/shopwareLabs/sw-cli-tools/releases" />
     </phar>
+    <phar alias="paratest" composer="brianium/paratest">
+        <repository type="github" url="https://github.com/paratestphp/paratest/releases" />
```

Comment:
That URL is likely to be wrong. It should be the api link. Can you fix that?

Reply:
Sorry, fixed.